### PR TITLE
Update content cli deploy less

### PR DESCRIPTION
**Pull request by mrtuvn**
## Purpose of this pull request

This pull request (PR) Update content to make more clarify

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.html
-  https://devdocs.magento.com/guides/v2.4/config-guide/cli/config-cli-subcommands-less-sass.html

## Links to Magento source code

-  ...

### CODE REVIEW

File: src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ Command options:
 bin/magento dev:source-theme:deploy [--type="..."] [--locale="..."] [--area="..."] [--theme="..."] [file1] ... [fileN]
 ```
 
+{:.bs-callout-info}
+This command create symlinks LESS files into `var/view_preprocessed` and `pub/static` folders during development. This process will not compile less files to css files.
```
Comment (BarnyShergold) [inline src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md:23]:
```suggestion
This command creates symlinks for LESS files in `var/view_preprocessed` and `pub/static` folders during development. This process will not compile LESS files to CSS files.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

File: src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md (hunk 1 above)
Comment (BarnyShergold) [reply src/guides/v2.3/config-guide/cli/config-cli-subcommands-less-sass.md:23]: Please also add a link the 2.3 version of the file in the PR description

**Review by BarnyShergold — APPROVED** [review]
all good!
You do realise that when change suggestions are made, you can just accept them rather than making changes locally and then pushing again?

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
Thanks, Tu!

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8834`.

**Pull request merged**